husky_components/navigation/main.js

```diff
@@ -433,8 +425,13 @@ define(function() {
          */
         bindDOMEvents: function() {
             this.sandbox.dom.on(this.$el, 'click', this.toggleItems.bind(this), '.navigation-items-toggle, .navigation-subitems-toggle');
-            this.sandbox.dom.on(this.$el, 'click', this.settingsClicked.bind(this), '.js-navigation-settings');
-            this.sandbox.dom.on(this.$el, 'click', this.selectSubItem.bind(this), '.js-navigation-sub-item, .js-navigation-item');
+
+            this.sandbox.dom.on(this.$el, 'click', function(event) {
+                event.preventDefault();
+                this.selectMenuItem(event.currentTarget);
+                _.delay(this.resizeListener.bind(this), 700); // TODO: is this really neccessary?
```

Comment:
I dont know. This was in the code before. I refactored a lot of the code. This piece moved from another section of the code. I didn't had time to check all issues i found, so I added a todo